index.html

```diff
@@ -22,7 +28,12 @@
 <body>
   <header id="header">
     <nav id="nav">  
-      <h1 id="title"><a href="/"><img src="./img/title.png" alt="판다마켓 타이틀"></a></h1>
+      <h1 id="title"><a class="title-logo" href="/">
+        <picture>
+          <source media="(max-width: 46.4rem)" srcset="./img/logo-text.png">
```

Comment:
HTML에서는 선택사항이지만, br, hr, img, input 등과 같은 빈 태그들은 self-closing 해주는 습관을 들이면 좋습니다. (XML 문법에서는 무조건 지켜야 하거든요...)

```suggestion
          <source media="(max-width: 46.4rem)" srcset="./img/logo-text.png" />
```